mkdocs.yml

```diff
@@ -373,9 +373,9 @@ nav:
             - "-XX:[+|-]JITServerLogConnections"                                 : xxjitserverlogconnections.md
             - "-XX:JITServerPort"                                                : xxjitserverport.md
             - "-XX:JITServerShareROMClasses"                                     : xxjitservershareromclasses.md
-            - "-XX:JITServerSSLCert"                                             : xxjitserversslcert.md
-            - "-XX:JITServerSSLKey"                                              : xxjitserversslkey.md
-            - "-XX:JITServerSSLRootCerts"                                        : xxjitserversslrootcerts.md
+            - "-XX:JITServerSSLCert"                                             : xxjitserversslcert.md             
+            - "-XX:JITServerSSLKey"                                              : xxjitserversslcert.md        #redirect
+            - "-XX:JITServerSSLRootCerts"                                        : xxjitserversslcert.md        #redirect 
```

Comment:
Please remove the trailing whitespace from these lines.

Reply:
Done, although I don't believe it will affect the docs. Is there another reason why trailing whitespace is undesirable?

Reply:
In some file formats it isn't significant, but in some contexts it can lead to hard-to-diagnose problems: in my opinion it's just safer to avoid them everywhere. In fact, openj9 and omr now have a check that pull requests don't introduce any new trailing whitespace.

Reply:
Interesting, I wasn't aware of that. I'll try to look out for trailing whitespace in future.